packages/core/test/dialog/dialogTests.tsx

```diff
@@ -192,6 +192,18 @@ describe("<Dialog>", () => {
             // test existence here because id is generated
             assert.notExists(dialog.find(".no-default-if-no-title").hostNodes().prop("aria-labelledby"));
         });
+
+        it("supports ref objects attached to container", () => {
+            const containerRef = React.createRef<HTMLDivElement>();
+            mount(
+                <Dialog containerRef={containerRef} usePortal={false}>
```

Comment:
you also have to render the dialog with `isOpen={true}` in order for the container & ref to get mounted. I have a fix, will push a commit